thermometers_humidity_sensors.py

```diff
@@ -19,54 +17,91 @@ def get_datas(self):
     def get_type(self):
         return self.type
 
+    def validate_sensor(self, reference_thermometer, reference_humidity):
```

Comment:
Should only use 1 reference.

Or you can create a new function to "set_reference" before validate.